inbox/events/remote_sync.py

```diff
@@ -49,7 +49,9 @@ def __init__(
     ):
         bind_context(self, "eventsync", account_id)
         self.provider = provider_class(account_id, namespace_id)
-        self.log = logger.new(account_id=account_id, component="calendar sync")
+        self.log = logger.new(
```

Comment:
This is just a small tweak so I can filter logs by calendar_sync for microsoft (and exclude Google). Mail sync greenlets already have provider set.